jenkins_jobs/modules/parameters.py

```diff
@@ -1112,6 +1112,36 @@ def git_parameter_param(registry, xml_parent, data):
     ]
     helpers.convert_mapping_to_xml(pdef, data, mapping, fail_required=True)
 
+def param_separator(registry, xml_parent, data):
+    """yaml: parameter-separator
+    A parameter separator.
+
+    :arg str name: UUID of the separator (optional)
+    :arg str separator-style: the style of the separator (optional)
+    :arg str section-header-text: the section header text of the separator (optional)
+    :arg str section-header-style: the section header style (CSS) of the separator. (optional)
+
+    Example::
+
+      parameters:
+        - parameter-separator:
+            name: lorem
+            separator-style: FOO
+            section-header: bar
+            section-header-style: font-weight:bold;z-index:10000
+    """
+    element_name = (
+        "jenkins.plugins.parameter__separator.ParameterSeparatorDefinition"
+    )
+    pdef = XML.SubElement(xml_parent, element_name)
```

Comment:
Don't we need an empty `<description/>` tag, even though it's not used?

Reply:
We don't need it, it does not have `description` tag.

Reply:
Well the old XML for binpkg does, right?